manifests/hash.pp

```diff
@@ -27,7 +27,7 @@
 define postfix::hash (
   Enum['present', 'absent']             $ensure='present',
   Variant[Array[String], String, Undef] $source=undef,
-  Variant[Array[String], String, Undef] $content=undef,
+  Optional[Variant[Sensitive[String],String]] $content = undef,
```

Comment:
I suspect that `Array` was used accidentally in the first place, but you'll want to get `Undef` back in there.

Reply:
Isn’t modern convention to use `Optional` instead of `Variant[Undef,...]`?

Reply:
sorry, misread :)